addon/components/course/objectives.hbs

```diff
@@ -0,0 +1,103 @@
+<section
+  class="detail-objectives {{if this.showCollapsible "collapsible"}}"
+  data-test-course-objectives
+  {{did-insert (perform this.load) @course @course.cohorts}}
+  {{did-update (perform this.load) @course @course.cohorts}}
+>
+  <div class="detail-objectives-header">
+    {{#if this.isManaging}}
+      <div class="title">
+        {{#if this.isManagingParents}}
+          <span class="specific-title">
+            {{t "general.objectiveParentTitleSingular"}}
+          </span>
+        {{/if}}
+        {{#if this.isManagingDescriptors}}
+          <span class="specific-title">
+            {{t "general.objectiveDescriptorTitle"}}
+          </span>
+        {{/if}}
+      </div>
+    {{else}}
+      <div
+        class="title {{if this.showCollapsible "clickable collapsible"}}"
+        role="button"
+        {{on "click" this.collapse}}
+      >
+        {{t "general.objectives"}} ({{@course.objectives.length}})
+      </div>
+    {{/if}}
+    <div class="detail-objectives-actions" data-test-actions>
+      {{#if this.isManaging}}
+        <button
+          type="button"
+          class="bigadd"
+          {{on "click" (perform this.save)}}
+          aria-label={{t "general.save"}}
+          data-test-save
+        >
+          <FaIcon @icon="check" />
+        </button>
+        <button
+          type="button"
+          class="bigcancel"
+          {{on "click" this.cancel}}
+          aria-label={{t "general.cancel"}}
+          data-test-cancel
+        >
+          <FaIcon @icon="undo" />
+        </button>
+      {{else if @editable}}
+        <LinkTo @route="course-visualize-objectives" @model={{@course}}>
```

Comment:
even if the course objectives are not editable by the current user - shouldn't they still be able to use the visualizations?

Reply:
They aren't right now https://github.com/ilios/common/blob/master/addon/components/detail-objectives.hbs#L37

Reply:
i accept this.